uaa/src/main/webapp/WEB-INF/spring-servlet.xml

```diff
@@ -210,11 +210,11 @@
                        key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(1)}"/>
                 <entry value-ref="utf8ConversionFilter"
                        key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(2)}"/>
-                <entry value-ref="corsFilter"
-                       key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(3)}"/>
                 <entry value-ref="limitedModeUaaFilter"
-                       key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(4)}"/>
+                       key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(3)}"/>
                 <entry value-ref="identityZoneResolvingFilter"
+                       key="#{T(org.cloudfoundry.identity.uaa.security.web.SecurityFilterChainPostProcessor.FilterPosition).position(4)}"/>
+                <entry value-ref="corsFilter"
```

Comment:
Rather than hard code 5 as position, do we want to make this corsFilter be `after` identityZoneResolvingFilter like I see being done below for some others? And should the corsFilter be after identityZoneSwitchingFilter (I don't know if the switching will result in the wrong zone being used by the cors filter).

Reply:
Good point.  Moved corsFilter position after IdentityZoneSwitchingFilter.  IdentityZoneSwitchingFilter allows to specify target zone using a header rather than subdomain.  It's not a common use case in Predix; however, it would be possible to bypass target zone cors policy if we apply corsFilter prematurely, before zone is switched.